tox.ini

```diff
@@ -52,9 +53,10 @@ deps =
 
     cov: coverage
 
-    astropy30: astropy==3.0.*
-    astropy40: astropy==4.0.*
-    astropylts: astropy==4.0.*
+    astropy3: astropy>=3.0.0,<4.0.0
+    astropy4: astropy>=4.0.0,<5.0.0
+    astropy5: astropy>=5.0.0,<6.0.0
+    astropylts: astropy>=5.0.0,<6.0.0
```

Comment:
Is this a typo, or willingly equal to astropy5?

Reply:
It was purely a copy & paste & update of the previous version — just bumping the LTS version from 4 to 5.

Since `lts` isn't actually being tested in CI, I wondered if it was useful at all... but I figured that was a question for another PR. :-)

Reply:
Actually, I think my definition here is incorrect — it should be `astropy>=5.0.0,<5.1.0`, so I'll fix tat.